pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[tool.poetry]
+name = "bitmapist"
+version = "3.107"
+description="Implements a powerful analytics library using Redis bitmaps."
+authors = [
+    "Amir Salihefendic <amix@amix.dk>",
+    "Doist Developers <dev@doist.com>",
+]
+license="BSD"
+readme = "README.md"
+homepage="http://www.amix.dk/"
```

Comment:
We should change this to another homepage 😅

Reply:
True! Any ideas which one? We could just drop it, since its optional, and the link to the git repo is probably good enough anyway?